apps/src/sites/code.org/pages/views/job_board.js

```diff
@@ -0,0 +1,59 @@
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import JobBoard from '@cdo/apps/templates/jobBoard/JobBoard';
+
+$(document).ready(showJobBoard);
+
+function showJobBoard() {
+  const jobBoardElement = $('#job-board-container');
+  const jobsByDepartment = {};
+
+  $.ajax({
+    type: 'GET',
+    url: 'https://boards-api.greenhouse.io/v1/boards/codeorg/jobs?content=true'
+  })
+    .done(result => {
+      // Reshape into an object with jobs by department. Shape:
+      // {
+      //   departmentName: "Engineering",
+      //   jobs: [
+      //           {
+      //             jobTitle: "Software Engineer"
+      //             location: "Seattle",
+      //             applicationUrl: "https://boards.greenhouse.io/codeorg/jobs/4098667005"
+      //           },...
+      //   ]
+      // }
+      result.jobs.forEach(job => {
+        const departmentName = job.departments[0].name;
```

Comment:
Not sure how robust this needs to be, but will there always be a `departments[0]` member?

Reply:
Good question, not 100% sure. As written, we need a department name to display jobs, so I can at least safe navigate here to avoid breaking things if its missing

Reply:
TJ confirmed all jobs should have a department, but added safe navigation